File: armstrong.js
function armstrong(num) {
    const numString = num.toString();
    const numDigits = numString.length;
    let sum = 0;

    for (let digits of numString) {
        sum += Math.pow(parseInt(digits), numDigits);
    }
    if (numString == sum) {
        document.write(numString + " is an Armstrong number <div>");
    }
    else {
        document.write(numString + " is not an Armstrong number <div>");
    }
}
armstrong(153)
armstrong(370)
armstrong(10)

File: armstrong.py
num = int(input("Enter any number: "))
sum = 0
# find the sum of the power of value of number of digits of each digit
temp = num
c = str(temp)
n = len(c)
while temp > 0:
    digit = temp % 10
    sum += digit ** n
    temp //= 10
if num == sum:
    print(num, " is an Armstrong number")
else:
    print(num, " is not an Armstrong number")